stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,433 @@
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe">    
+            </div>
+         
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    <span>Maecenas convallis velit quis felis dictum, in ultrices quam faucibus.        
+                </p>
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 22, 2020</small>
+                </div>
+            </div>     
+        </div>
+        
+        <div class="fd-feed-list__body">
+            <div
+                class="fd-avatar fd-avatar--s fd-feed-list__thumb"
+                style="background-image: url(http://lorempixel.com/460/620/people/9/)" 
+                role="img" 
+                aria-label="John Doe"
+                title="John Doe"
+            ></div> 
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    <span class="fd-feed-list__name">Sanchez: </span>
+                    <span>Maecenas convallis velit quis felis dictum, in ultrices quam faucibus.        
+                </p>
+                <div class="fd-feed-list__footer">
+                    <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                </div>
+            </div>
+       </div>
+    </div>
+</div>
+`;
+
+feedListGroup.storyName = 'Group';
+feedListGroup.parameters = {
+    docs: {
+        storyDescription: 'Group'
+    }
+};
+
+export const small = () => `
+    <div class="fd-feed-list fd-feed-list--s" aria-label="Feed List Item small size example">
+        <div class="fd-feed-list__body">
+            <div style="display: flex">
+                <div
+                    class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list__thumb" 
+                    style="background-image: url(http://lorempixel.com/460/620/people/7/)" 
+                    role="img" 
+                    aria-label="John Doe"
+                    title="John Doe">    
+                </div>
+                <div>
+                    <span class="fd-feed-list__name">Joe Doe: </span>
+                    <div class="fd-feed-list__footer">
+                        <small class="fd-feed-list__footer--date">Aug 20, 2020</small>
+                    </div>
+                </div>
+            </div>    
+        
+            <div class="fd-feed-list__content">
+                <p class="fd-feed-list__text">
+                    Suspendisse facilisis sed odio in mollis. Donec quis justo aliquam, porta justo a, accumsan lorem.
+                </p>
+            </div>
+        </div> 
+    </div>
+`;
+
+small.storyName = 'Small';
+small.parameters = {
+    docs: {
+        storyDescription: 'Mobile Responsive'

```

Comment:
The feed list item can be mobile responsive by adding the `fd-feed-list--s` modifier class to the main element.